sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dmac_training.py

```diff
@@ -11,8 +11,7 @@
 from azure.core.credentials import AzureKeyCredential
 from azure.core.exceptions import ClientAuthenticationError, HttpResponseError
 from azure.ai.formrecognizer._generated.v2022_06_30_preview.models import GetOperationResponse, ModelInfo
-from azure.ai.formrecognizer._models import CustomFormModel, DocumentModelInfo
```

Comment:
Snuck in some fixes for these weird imports in these test files for DocumentModelInfo...I'll do a wider check in the tests later, but decided to tackle these now since I was already touching these files.